src/dune_engine/build_config.mli

```diff
@@ -56,6 +56,22 @@ module Error : sig
     val to_dyn : t -> Dyn.t
   end
 
+  module Event : sig
+    type nonrec t =
+      | Add of t
+      | Remove of t
+  end
+
+  module Set : sig
+    type nonrec t =
+      { current : t Id.Map.t
+      ; stamp : int
```

Comment:
Could you add a comment explaining the purpose of the `stampt`?

Reply:
I improve this module a bunch. Now all of this stamp stuff is completely private. So there's no need to explain this somewhat error prone and confusing stuff.

Reply:
Awesome!